containers/RoundSection/index.js

```diff
@@ -67,13 +25,80 @@ const RoundSection = ({
   btnStyle,
   sectionSubTitle,
   cardArea,
-  btnTitle,
+  btnTitle
 }) => {
+  const [hasRound, setHasRound] = useState(true);
```

Comment:
Since we have 3 states, easier to use `useReducer` to merge the states instead of manipulating it one by one.

```
const [state, setState] = useReducer((s, a) => ({ ...s, ...a }), {
    hasRound: false, // this should be false by default too, since no round is more likely?
    isLoading: true,
    endTime: 0,
  });
```